webapi/src/main/scala/org/knora/webapi/responders/admin/UsersResponderADM.scala

```diff
@@ -1504,6 +1439,78 @@ class UsersResponderADM(responderData: ResponderData) extends Responder(responde
     } yield UserOperationResponseADM(updatedUserADM.ofType(UserInformationTypeADM.RESTRICTED))
   }
 
+  /**
+    * Updates the password for a user.
+    *
+    * @param userIri              the IRI of the existing user that we want to update.
+    * @param password             the new password.
+    * @param featureFactoryConfig the feature factory configuration.
+    * @param requestingUser       the requesting user.
+    * @param apiRequestID         the unique api request ID.
+    * @return a future containing a [[UserOperationResponseADM]].
+    * @throws BadRequestException         if necessary parameters are not supplied.
+    * @throws UpdateNotPerformedException if the update was not performed.
+    */
+  private def updateUserPasswordADM(userIri: IRI,
+                                    password: Password,
+                                    featureFactoryConfig: FeatureFactoryConfig,
+                                    requestingUser: UserADM,
+                                    apiRequestID: UUID): Future[UserOperationResponseADM] = {
+
+    log.debug("updateUserPasswordADM - new password: {}", password.value)
```

Comment:
isn't it a bit risky to log the password like this? In a real life scenario, a user's password would end up somewhere in the docker logs.

Reply:
Absolutely, this is not good at all. It this was there before, I deleted this line now.